x-pack/plugins/task_manager/server/saved_objects/migrations.test.ts

```diff
@@ -108,6 +108,26 @@ describe('successful migrations', () => {
       });
     });
   });
+
+  describe('8.2.0', () => {
+    test('resets attempts and status of the failed alerting tasks without schedule interval', () => {
```

Comment:
Can we add a test to verify that tasks with status != failed and no schedule do not get migrated?

Reply:
Done :)